README.md

```diff
@@ -1,6 +1,148 @@
-# curriculum-cr-training-mobile-menu
 
+<!-- TABLE OF CONTENTS -->
 
-This repo host an initial code for the tasks performed during the Code Reviewers training.
+# 📗 Table of Contents
 
-Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  <!-- - [🚀 Live Demo](#live-demo) -->
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Developer portfolio <a name="about-project"></a>
+
+This is a portfolio page to showcase the skils of a developer through the applications built
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Tech Stack</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+    <li><a href="https://www.w3.org/Style/CSS/Overview.en.html">CSS</a></li>
+    <li><a href="https://www.javascript.com/">JS</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Showcase form**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- Html & CSS basics
+- JavaScript
+- Git & github
+
+### Clone Repository
+
+Clone the repository using the following bash command in an appropriate location.
+
+```bash
+  git clone git@github.com:shahadat3669/training-mobile-menu.git
+```
+Go to the project directory.
+
+```bash
+  cd training-mobile-menu
+```
+
+
+### Install
+
+Install this project with:
+- Once oponed up in a code editor run `npm install` to install all packages
+
+
+### Usage
+
+- Survey form.
+
+
```

Comment:
- [ ] I noticed this line "- Survey form." in the README file appears to be empty or incomplete, Please remove it from the README file. Including incomplete or unnecessary information can lead to confusion for users and contributors. By removing this line, we can ensure that the README file remains clear and concise, providing only relevant and complete sections to users and contributors.

Reply:
Kindly implement this requested change by your previous reviewer. You should add complete information to your documentation. Because we do not want the users to be confused 🫨🫨🫨 The essence of documentation is to help users to use our product 🧐